frontend/ui/widget/imagewidget.lua

```diff
@@ -165,12 +165,29 @@ function ImageWidget:_render()
 
     -- First, rotation
     if self.rotation_angle ~= 0 then
-        if not self._bb_disposable then
-            -- we can't modify _bb, make a copy
-            self._bb = self._bb:copy()
+        -- Allow for easy switch to former scaling via blitbuffer methods
+        if G_reader_settings:isTrue("use_legacy_image_scaling") then
```

Comment:
I think just `legacy_image_scaling` would suffice. If a setting is necessary at all.